static/css/main.css

```diff
@@ -0,0 +1,36 @@
+body {
+    padding: 0;
+    margin: 0; 
+    background: #f2f6e9;
```

Comment:
please maintain consistent code alignment so its easier for others to read it

Reply:
OK! i did it, I upload the  code when I finish the  base and child templates :)